nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs

```diff
@@ -142,7 +142,7 @@ public void SetData(TViewModel model, bool isConsumable)
             }
             else
             {
-                row = Game.Game.instance.TableSheets.ItemSheet.Values
+                row = Game.Game.instance.TableSheets.EquipmentItemSheet.Values
```

Comment:
This change make some problems I guess. The `ItemView` module support not only equipment.

Reply:
This method is used only in combination, but I think i should extend this to support both equipment and other item. Gonna fix it!

Reply:
The `EqiupmentItemSheet` does not contains any of `Consumable` rows. Is it OK?

Reply:
```
            if (isConsumable)
            {
                row = Game.Game.instance.TableSheets.ConsumableItemSheet.Values
                    .FirstOrDefault(r => r.Id == model.ItemBase.Value.Id);
            }
            else
            {
                row = Game.Game.instance.TableSheets.EquipmentItemSheet.Values
                    .FirstOrDefault(r => r.Id == model.ItemBase.Value.Id);
            }
 ```
 
 It is already handled like this.

Reply:
!!!! Oh I missed. 😅